spec/models/polling/cert_status_updater_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+require 'polling/cert_status_updater'
+
+RSpec.describe CertStatusUpdater, type: :model do
+
+  let(:root) { PKI.new }
+
+  let(:msa_enc_cert_1) { create(:msa_encryption_certificate, in_use_at: nil) }
```

Comment:
we don't need to explicitly set the in_use_at, it's already done in the factory
```suggestion
  let(:msa_enc_cert_1) { create(:msa_encryption_certificate) }
```

Reply:
It's an important part of the test so I am writing it explicitly.  This also means the tests will still do what they are supposed to do even if we later change the behaviour of the factory.